### ISSUE TITLE
Feature Request: Expose amount of native asset locked up by min balance, trustlines, etc

### ISSUE THREAD

**Issue by leighmcculloch** [stellar/go#2684]
### What problem does your feature solve?
It's difficult to determine how much native balance in an account is locked up due to the account minimum balance and additional subentries on the account.

### What would you like to see?
A field added to the account exposing the lumens locked up, either as a single value alone, or as a total value along with an array of descriptors and values that can be sum'd describing what each item is that locks up the amount.

### What alternatives are there?
Calculating the value manually is always an option.

**Comment by bartekn** [comment]
I can see pros and cons of adding this.

Advantages:
* Having this value in Horizon can be helpful when the method used to calculate locked balance changes in the future (no need to update client code).
* It required a really small amount of time to add this.

Disadvantages:
* The value can be easily calculated using the current account response.
* It can be confusing: Should we also include a base fee for the transaction to move all available lumens or not? If so, what if surge pricing is activated?
* It can change after user reads it: someone can send lumens to that account or an active offer selling/buying XLM is consumed.

Would love to hear others' thoughts about this.

**Comment by leighmcculloch** [comment]
>The value can be easily calculated using the current account response.

**Comment by leighmcculloch** [comment]
>The value can be easily calculated using the current account response.

This is true today, but will not be true if CAP-33 is implemented. Once it is implemented it will be quite difficult to calculate the amount without making multiple calls to Horizon and so adding this I think would insulate Horizon from unnecessary requests.

>Should we also include a base fee for the transaction to move all available lumens or not?

I don't think we need to include any base fee for transactions. The value should capture only the amount of lumens locked up and unavailable for use at the end of the last ledger.

>It can change after user reads it

This is one of the reasons I think we need this. The value changes, it evolves, and some wallets display this alongside the balance which means they might poll this on some frequency along with the balance. Once CAP-23 and CAP-33 is out this will mean wallets polling many different endpoints to accurately reflect the lumens that are locked up. If the value is displayed alongside the accounts balance it will only require hitting that one endpoint.

**Comment by mollykarcher** [comment]
Thank you for submitting this feature request.

At this time, we are not accepting new feature requests for Horizon, and we’re focusing our efforts on maintenance, stability, and existing functionality. As such, we are closing this issue.

We appreciate your interest and the time you took to share your ideas.
